proto/service/metadata/types/metadata.proto

```diff
@@ -75,6 +101,14 @@ message GetUserBucketsCountRequest {
   string account_id = 1;
 }
 
+// ListExpiredBucketsBySpRequest is the request type for the ListExpiredBucketsBySp RPC method.
+message ListExpiredBucketsBySpRequest {
+  // create_at defines the create timestamp of bucket
+  int64 create_at = 1;
+  // primary_sp_address defines the primary sp address of bucket
+  string primary_sp_address = 2;
```

Comment:
Don't support pagination?  If there are a large number of expired buckets, will returned at once?

Reply:
If no pagination, can we add another parameter `limit`, so we can limit the result and always get the oldest buckets for stop serving?

Reply:
Discussed and added a new parameter limit, which user can pass in 1 - 1000 as limit. Other limit input will be defaulted to 1000.

Reply:
BTW, I noticed that other API like `ListObjectsByBucketName` didn't support pagination or have a limit number. Could you add a todo to fix these. @annielz